lib/public/components/qcFlags/qcFlagsPagesButtons.js

```diff
@@ -57,15 +59,26 @@ export const qcFlagCreationPanelLink = (
         Success: (detectors) => {
             const userDetectors = detectors.find(({ id }) => id === dplDetectorId);
```

Comment:
I know this is old code, but I think it would be more clear to name the variable `userHasAccessToDetector`

Reply:
yes, renamed

Reply:
My message was for the variable `userDetectors` sorry if this was not clear (the change you did is good too)

Reply:
no, you put it clear, I've read to fast, also renamed